var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -15,14 +15,30 @@ class Llvm(CMakePackage):
     """
 
     homepage = 'http://llvm.org/'
-    url = 'http://llvm.org/releases/3.7.1/llvm-3.7.1.src.tar.xz'
+    url      = "https://github.com/llvm/llvm-project/archive/llvmorg-7.1.0.tar.gz"
     list_url = 'http://releases.llvm.org/download.html'
 
     family = 'compiler'  # Used by lmod
 
-    # currently required by mesa package
-    version('3.0', 'a8e5f5f1c1adebae7b4a654c376a6005',
-            url='http://llvm.org/releases/3.0/llvm-3.0.tar.gz')
+    version('develop', git='https://github.com/llvm/llvm-project')
```

Comment:
Spack now has the ability to list both `url` and `git` at the top-level. Can you move this `git` next to `url` above? Then just add `branch='master'`. I would actually rename this version to `master` now that #1983 has been merged.